yarn.lock

```diff
@@ -1211,7 +1211,7 @@
 
 "@contentful/rich-text-types@^16.6.1":
   version "16.8.3"
-  resolved "https://npm.pkg.github.com/download/@contentful/rich-text-types/16.8.3/f0effea5d0ed23f9abfd992c9b1be8a00f4d63e9#f0effea5d0ed23f9abfd992c9b1be8a00f4d63e9"
+  resolved "https://registry.npmjs.org/@contentful/rich-text-types/-/rich-text-types-16.8.3.tgz#f0effea5d0ed23f9abfd992c9b1be8a00f4d63e9"
```

Comment:
This fixes the symptom, but not necessarily the cause. If new packages are added from the `@contentful` scope, it's likely we'll see this happen again. Is it maybe worth explicitly adding config so that Yarn knows not to use GHPR? E.g. adding `.npmrc` with `@contentful:registry=https://registry.npmjs.org` would be enough.

Also - why NPM registry here when the rest of the lockfile is using `registry.yarnpkg.com`? They're a CNAME from memory anyway, but 🤷

Reply:
Approving as this isn't blocking the merge - we can follow up

Reply:
Added `.npmrc` 🙂